spring-cloud-stream-binder-kafka-docs/src/main/asciidoc/dlq.adoc

```diff
@@ -1,24 +1,26 @@
 [[kafka-dlq-processing]]
 == Dead-Letter Topic Processing
 
-Because it can't be anticipated how users would want to dispose of dead-lettered messages, the framework does not provide any standard mechanism to handle them.
+Because you cannot anticipated how users would want to dispose of dead-lettered messages, the framework does not provide any standard mechanism to handle them.
```

Comment:
anticipate?

Reply:
Yessir. I make typoes, too, which is why I appreciate reviews. Thanks. :)